bin/test-entry-creation.js

```diff
@@ -0,0 +1,36 @@
+require('dotenv').config();
+
+const { createNewEntry, publishEntry } = require('../lib/content');
+const { waitUntilContentIsDelivered } = require('../lib/delivery');
+
+const cmaToken = process.env.CMA_TOKEN;
+const spaceId = process.env.SPACE_ID;
+const cdaToken = process.env.CDA_TOKEN;
+
+run();
```

Comment:
Let's catch errors that could appear during execution, otherwise unhandled promise rejection.

Reply:
Not sure if that's a good idea - don't we want it to fail and stop when we got an error ?